docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -691,6 +691,18 @@ Start Jenkins
                user: your.chosen.username
                password: your.chosen.password
 
+11. In a local setup, you have to disable CSRF otherwise some API endpoints will return HTTP Status 403 Forbidden.
```

Comment:
Note: this is not new, but got lost when removing the secret push token section in #9266.